server/src/main/java/com/datastrato/gravitino/server/GravitinoServer.java

```diff
@@ -81,9 +79,13 @@ private void initializeRestApi() {
           protected void configure() {
             bind(gravitinoEnv.metalakesManager()).to(MetalakeManager.class).ranked(1);
             bind(gravitinoEnv.catalogManager()).to(CatalogManager.class).ranked(1);
+
             if (enableAuthorization) {
-              bind(gravitinoEnv.accessControlManager()).to(AccessControlManager.class).ranked(1);
+              bind(Optional.of(gravitinoEnv.accessControlManager())).to(Optional.class).ranked(1);
```

Comment:
Can't you use `Optional.ofEmpty()`?

Reply:
Also what is the purpose of this change?

Reply:
> Can't you use `Optional.ofNullable()`?

Yes, I can.

Reply:
> Also what is the purpose of this change?

Because if we use name binding of jersey filter, the jersey will resolve the Operations. If we pass a null parameter, it will throw an exception because of injection failure.